Tweak description for diagnostic plugins (#335)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/configure-diagnostic-logging.adoc

= Configure diagnostic logging

> :description: Hazelcast provides powerful diagnostic logging capabilities that generate comprehensive insights for troubleshooting and monitoring.

Reviewer: ```suggestion
:description: Hazelcast provides diagnostic logging capabilities to help with monitoring and troubleshooting.
```
Let's keep the marketing hyperbole out of tech docs please
